src/Runner.Worker/StepsRunner.cs

```diff
@@ -103,6 +104,17 @@ public async Task RunAsync(IExecutionContext jobContext)
                     // Set GITHUB_ACTION
                     step.ExecutionContext.SetGitHubContext("action", actionStep.Action.Name);
 
+                    if (ActionCommandManager.StepSummaryEnabled(step.ExecutionContext))
```

Comment:
Should we follow the pattern we have for https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable which is under `src/Runner.Worker/FileCommandManager.cs`?

Reply:
Following this pattern would handle all the file creation/deletion per step for you, so its a good idea 👍

Reply:
@TingluoHuang Can you elaborate on which pattern you are referring to. I might be missing some context here. :)

Reply:
We talked about this offline and decided to implement a new file command like @TingluoHuang suggested 👍